docs/configuration.md

```diff
@@ -439,6 +442,9 @@ The `clear-text` reporter supports three additional config options:
 - `allowColor` to use cyan and yellow in printing source file names and positions. This defaults to `true`, so specify as `clearTextReporter: { allowColor: false },` to disable if you must.
 - `logTests` to log the names of unit tests that were run to allow mutants. By default, only the first three are logged. The config for your config file is: `clearTextReporter: { logTests: true },`
 - `maxTestsToLog` to show more tests that were executed to kill a mutant when `logTests` is true. The config for your config file is: `clearTextReporter: { logTests: true, maxTestsToLog: 7 },`
+- `reportAllTests` to log all tests. This defaults to `true`, so specify as `clearTextReporter: { reportAllTests: false },` to disable if you must.
+- `reportAllMutants` to log all mutants. This defaults to `true`, so specify as `clearTextReporter: { reportAllMutants: false },` to disable if you must.
+- `reportScoreTable` to log score table. This defaults to `true`, so specify as `clearTextReporter: { reportScoreTable: false },` to disable if you must.
```

Comment:
The defaults can already be viewed above. I also rephased it a bit so it is more clear what to expect (I hope).

```suggestion
- `reportTests` to log tests. When set to `true` this reporter will report a list of all tests with the amounts of mutants they killed (if any).
- `reportMutants` to log mutants. When set to `true` this reporter will report each `Survived` and `NoCoverage` mutant to the stdout, as well as log other mutant states on debug log level.
- `reportScoreTable` to log the score table. When set to `true` this reporter will end with a summary table with a row per mutated file.
```

Reply:
Those descriptions look great, will update to them!